### DISCUSSION TITLE
Failed Building static HTML for pages

### DISCUSSION THREAD

**Discussion by Amiinoz** [gatsbyjs/gatsby#33669] [Help]
Hello everyone, I am having issues building my gatsby project, My project works fine with gatsby develop, however fails in gatsby build. I have worked on this project for a month and will be forever grateful for any help.

My initial gatsby build fail:  failed with node-fetch error.

What I have tried: gatsby-node.js

exports.onCreateWebpackConfig = ({ stage, loaders, actions }) => {
  if (stage === "build-html" || stage === "develop-html") {
    actions.setWebpackConfig({
      module: {
        rules: [
          {
            test: /firebase/,
            use: loaders.null(),
          },
        ],
      },
    })
  }
}

My current gatsby build fail:
<img width="894" alt="gatsbyBuildFail" src="https://user-images.githubusercontent.com/20507343/138758971-b4b313e3-efb4-47c7-90c2-899757edee23.png">

My system info:
<img width="505" alt="gatsbyBuildError2" src="https://user-images.githubusercontent.com/20507343/138759311-0543fd70-1804-46ec-aaf3-8fb4d935f19c.png">

My dependencies 

<img width="536" alt="packageJson" src="https://user-images.githubusercontent.com/20507343/138760132-5e6c7dff-116c-4f80-9079-d46d4f184380.png">

**Comment by LekoArts** [comment]
Hi!

It currently fails on `locomotive-scroll`, see https://github.com/locomotivemtl/locomotive-scroll/issues/13 for help. More general help for this error: https://www.gatsbyjs.com/docs/debugging-html-builds/

**Comment by Amiinoz** [comment]
Great, thanks for the tip
